iogt_content_migration/management/commands/load_v1_db.py

```diff
@@ -270,41 +281,27 @@ def create_home_page(self, root):
 
         return home
 
-    def create_index_pages(self, homepage):
-        self.section_index_page = models.SectionIndexPage.objects.first()
-        if self.section_index_page is None:
-            self.section_index_page = models.SectionIndexPage(title='Sections')
-            homepage.add_child(instance=self.section_index_page)
-
-        self.banner_index_page = models.BannerIndexPage.objects.first()
-        if self.banner_index_page is None:
-            self.banner_index_page = models.BannerIndexPage(title='Banners')
-            homepage.add_child(instance=self.banner_index_page)
-
-        self.footer_index_page = models.FooterIndexPage.objects.first()
-        if self.footer_index_page is None:
-            self.footer_index_page = models.FooterIndexPage(title='Footers')
-            homepage.add_child(instance=self.footer_index_page)
-
-        self.poll_index_page = PollIndexPage.objects.first()
-        if self.poll_index_page is None:
-            self.poll_index_page = PollIndexPage(title='Polls')
-            homepage.add_child(instance=self.poll_index_page)
-
-        self.survey_index_page = SurveyIndexPage.objects.first()
-        if self.survey_index_page is None:
-            self.survey_index_page = SurveyIndexPage(title='Surveys')
-            homepage.add_child(instance=self.survey_index_page)
-
-        self.quiz_index_page = QuizIndexPage.objects.first()
-        if self.quiz_index_page is None:
-            self.quiz_index_page = QuizIndexPage(title='Quizzes')
-            homepage.add_child(instance=self.quiz_index_page)
-
-        self.miscellaneous_index_page = models.MiscellaneousIndexPage.objects.first()
-        if self.miscellaneous_index_page is None:
-            self.miscellaneous_index_page = models.MiscellaneousIndexPage(title='Miscellaneous')
-            homepage.add_child(instance=self.miscellaneous_index_page)
```

Comment:
Is there a reason to remove these safeguards against duplicate index pages?

Reply:
we delete all the index pages before content migration, so there is no need for this check